src/Pages/RoomDetail/BottomInfo/ReviewTab/EachReview/EachReview.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+import Theme from '../../../../../Styles/Theme';
+
+const EachReview = ({ review }) => {
+  const { icon, title, star, rate, option, contents, date, image } = review;
+  return (
+    <ReviewWrapper key={title}>
+      <ImgContainer>
+        <img src={icon} alt="icon img" />
+      </ImgContainer>
+      <ContentsContainer>
+        <div className="review_title">{title}</div>
+        {/* <span>
+          {star === 5 && (
+            <>
+              <i class="fas fa-star"></i>
+              <i class="fas fa-star"></i>
+              <i class="fas fa-star"></i>
+              <i class="fas fa-star"></i>
+              <i class="fas fa-star"></i>
+            </>
+          )}
+        </span> */}
+        <span className="rate">{rate.toFixed(1)}</span>
+        <div className="option_name">{option}</div>
+        <p className="contents">{contents}</p>
+        {/* 이미지를 올린 경우에만 이미지 추가 */}
+        <ImgWrapper isImgUploaded={image}>
+          {image &&
+            image.map(img => {
+              return <img src={img} alt="review img" />;
+            })}
+        </ImgWrapper>
+        <div className="dates_passed">{date}</div>
+      </ContentsContainer>
+    </ReviewWrapper>
+  );
+};
+
+export default EachReview;
+
+const ReviewWrapper = styled.li`
+  display: flex;
+  align-items: flex-start;
+  padding: 20px;
+  border-bottom: 1px solid ${Theme.borderGray};
+  img {
+    width: 55px;
+    height: 55px;
+  }
+`;
+
+const ImgContainer = styled.div`
+  margin-right: 20px;
+`;
+const ContentsContainer = styled.div`
```

Comment:
```suggestion
const ImgContainer = styled.div`
  margin-right: 20px;
`;

const ContentsContainer = styled.div`
```
구분되는 요소들끼리는 한줄 띄워주세요